e-commerce-ui/src/app/app.component.scss

```diff
@@ -0,0 +1,80 @@
+body::before {
+    content: ""; /* adding a blank section on top to make website more colorful */
+    display: block; /* should look like a block */
+    width: 100%; 
+    height: 50px; 
+    background-color: #0d4c76; 
+    position: absolute; 
+    top: 0;
+    left: 0;
+    z-index: 0; 
+}
+
+
+body {
+    font-family: 'Arial';
+    background-color: #f7f9fc;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    margin: 0;
+    color: #4a4a4a;
+    position: relative;
+}
+
+.form-container {
+    background-color: #ffffff;
+    padding: 40px 50px;
+    border-radius: 20px;
+    box-shadow: 0 30px 45px rgb(0, 0, 0);
+    width: 100%;
+    max-width: 400px;
+}
+
+h2 {
+    font-size: 24px;
+    margin-bottom: 30px;
+    font-weight: 500;
+    color: #333;
+    text-align: center;
+}
+
+
+input[type="text"],
+input[type="email"] {
+    width: 100%;
+    padding: 14px 16px;
+    border: 1px solid #e4e7eb;
+    border-radius: 5px;
+    font-size: 15px;
+    outline: none;
+    box-sizing: border-box;
+    transition: border 0.3s ease;
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+
+button {
+    width: 90%;
+    padding: 15px 20px;
+    border: none;
+    background-color: #072b4f;
+    color: #ffffff;
+    font-size: 16px;
+    border-radius: 5px;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+    outline: none;
+    margin-top: 20px;
+}
+
+.button-group {
+    display: flex;
+    justify-content: center;  /* horizontally center the button */
+    align-items: center;      /* vertically center the button */
+}
+
+button:hover {
+    background-color: #4a8ad6;
+}
```

Comment:
Learn how to use [Angular Materials Input](https://material.angular.io/components/input/overview).